README.md

```diff
@@ -1219,52 +1250,11 @@ start with the prefix `--terragrunt-`. The currently available options are:
   into it.
 
 
-### Terragrunt config files
-
-The current version of Terragrunt expects configuration to be defined in a `terraform.tfvars` file. Previous
-versions defined the config in a `.terragrunt` file. **The `.terragrunt` format is now deprecated**!
-
-For backwards compatibility, Terragrunt will continue to support the `.terragrunt` file format for a short period of
-time. Check out the next section for how this works. Note that you will get a warning in your logs every time you run
-Terragrunt with a `.terragrunt` file, and we will eventually stop supporting this older format, so we recommend
-migrating to the `terraform.tfvars` format ASAP!
-
-
-#### Config file search paths
+### Configuration
```

Comment:
Done.

I manually checked all of the links in the README and all appear to be working as expected.